File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        ans = 0
        count = 0

        nums.append(0)
        for x in nums:
            if x==0:
                ans = max(ans,count)
                count = 0
            else:
                count+=1
        
        return ans

        

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums){
        int ans = 0,count = 0;

        for(int x:nums){
            if(x==1)count+=1;
            if(x==0)count=0;
            if(ans<count)ans=count;
        }
        return ans;
    }
};